src/sass/components/_intro-story.scss

```diff
@@ -0,0 +1,79 @@
+.intro-story {
+  background-color: $gray-dark;
+  color: #fff;
+  text-align: center;
+  padding: 0;
+  height: 500px;
+  width: 100%;
+  @include breakpoints (small large){
```

Comment:
@sarah-clements FYI, this works, but consider the following...
This query scopes to min-width: [small], max-width: [large], which in this case is excessive. While you are unlikely to have a device/screen narrower than `small` (320px), there is no reason to constrain the media query to that min-width. You could simplify by simply declaring `large` (959px) to be the max-width (`breakpoints(max large)`)
...but also...
consider forming media queries (by default) so that the smaller screens get the default styles and then they are overridden by styles for larger screens. This is not important in this specific line, but the practice has other implications that make it a good practice.
So instead of setting `width: 500px;` as the default, consider dong this instead:
```
width: 100vh;
@include breakpoints(large){ // min-width: 960px
  width: 500px;
}
```
Make (at least some) sense?